libsql-sqlite3/src/loadext.c

```diff
@@ -517,12 +517,7 @@ static const sqlite3_api_routines sqlite3Apis = {
   sqlite3_stmt_explain
 };
 
-static const libsql_api_routines libsqlApis = {
```

Comment:
that's a delicate change since libsqlApis is public. If we want to nuke them it's better to do it now rather than later, but we need to clearly document and announce that, in case we have external users of the virtual WAL api.